public/locales/en-US/send.ftl

```diff
@@ -19,66 +19,70 @@ notifyUploadDone = Your upload has finished.
 
 uploadingPageMessage = Once your file uploads you will be able to set expiry options.
 uploadingPageCancel = Cancel upload
-  .title = {uploadingPageCancel}
+  .title = Cancel upload
 uploadCancelNotification = Your upload was cancelled.
 
 uploadingPageLargeFileMessage = This file is large and may take awhile to upload. Sit tight!
 uploadingFileNotification = Notify me when the upload is complete.
 uploadSuccessConfirmHeader = Ready to Send
 
-uploadSvgAlt.alt = Upload
-
-
-//Note the spec suggests that this string is editable. That feature will not appear at Launch
+uploadSvgAlt
+    .alt = Upload
 uploadSuccessTimingHeader = The link to your file will expire after 1 download or in 24 hours.
 copyUrlFormLabel = Copy and share the link to send your file:
 copyUrlFormLabelWithName = Copy and share the link to send your file: { $filename }
+
 // Note: Title text for button should be the same
 copyUrlFormButton = Copy to clipboard
-  .title = {copyUrlFormButton}
+    .title = Copy to clipboard
 
 copiedUrl = Copied!
 
 deleteFileButton = Delete file
-  .title = {deleteFileButton}
+    .title = Delete file
 sendAnotherFileLink = Send another file
-  .title = {sendAnotherFileLink}
+    .title = Send another file
 
-downloadAltText.alt = Download
+downloadAltText
+    .alt = Download
 downloadFileName  = Download { $filename }
 downloadFileSize = ({ $size })
 downloadMessage = Your friend is sending you a file with Firefox Send, a service that allows you to share files with a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
 downloadButtonLabel = Download
-  .title = {downloadButtonLabel}
+    .title = Download
 downloadNotification = Your download has completed.
 downloadFinish = Download Complete
 
 sendYourFilesLink = Try Firefox Send
-  .title = {sendYourFilesLink}
+    .title = Try Firefox Send
 downloadingPageProgress = Downloading { $filename } ({ $size })
 downloadingPageMessage = Please leave this tab open while we fetch your file and decrypt it.
 
-errorAltText.alt = Upload error
+errorAltText
+    .alt = Upload error
 errorPageHeader = Something went wrong!
 errorPageMessage = There has been an error uploading the file.
 errorPageLink = Send another file
 
 
-linkExpiredAlt.alt = Link expired
+linkExpiredAlt
+    .alt = Link expired
 expiredPageHeader = This link has expired or never existed in the first place!
 notSupportedHeader = Your browser is not supported.
 notSupportedDetail = Unfortunately this browser does not support the web technology that powers Firefox Send. You'll need to try another browser. We recommend Firefox!
```

Comment:
Nit: proper apostrophe instead of straight quote.

Note about Firefox Send if needed.